style.css

```diff
@@ -320,3 +338,252 @@ nav {
   margin-top: 50px;
   margin-bottom: 10px;
 }
+
+.snap01 {
+  display: none;
+}
+
+.list-1 {
+  list-style: none;
+}
+
+.list-item-1 {
+  display: flex;
+  justify-content: space-between;
+}
+
+.list-2 {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100px;
+  list-style: none;
+}
+
+.list-item-2 {
+  background: #f7f7f9;
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  margin-top: 20px;
+  border-radius: 5px;
+}
+
+.name {
+  margin-left: 10px;
+}
+
+.list-3 {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 30px;
+}
+
+.list-4 {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+@media screen and (min-width: 1024px) {
+  .logo {
+    margin-left: 60px;
+    margin-top: 10px;
+  }
+
+  .mobile-nav {
+    display: none;
+  }
+
+  .desktop-nav {
+    display: inherit;
+    margin-right: 30px;
+  }
+
+  .nav-link {
+    text-decoration: none;
+    margin-right: 30px;
+    color: #344563;
+  }
+
+  .homepage {
+    height: 768px;
+    background-image: url(./images/background_1024.png);
+  }
```

Comment:
- [ ] Consider making your `.homepage` design to match that of your chosen template for the desktop and mobile version by adding the correct `border-radius`.

Your design: 
<img width="932" alt="chrome_eHV3l8BgXx" src="https://user-images.githubusercontent.com/81468675/144471470-0f4d2d20-c5de-4c6f-9be8-bb7732886b7c.png">


Figma design:
<img width="269" alt="chrome_7Rb7HZpjwm" src="https://user-images.githubusercontent.com/81468675/144471432-eb857163-9650-49c6-b931-9a031e2a7ac3.png">